docker/dockerfile-hcc-lc-ubuntu-16.04

```diff
@@ -6,7 +6,7 @@ RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install --no-instal
   curl -sL http://packages.amd.com/rocm/apt/debian/rocm.gpg.key | apt-key add - && \
   sh -c 'echo deb [arch=amd64] http://packages.amd.com/rocm/apt/debian/ xenial main > /etc/apt/sources.list.d/rocm.list' && \
   apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install --no-install-recommends -y \
-  hsa-rocr-dev g++-multilib gcc-multilib findutils libelf1 libpci3 file && \
+  hsa-rocr-dev g++-multilib gcc-multilib findutils libelf1 libpci3 file libunwind-dev && \
```

Comment:
Instead of manually installing libunwind-dev to the docker install image, would it work better to make it a dependency on the debian package so that users benefit from package management installing the dependency?  I suppose same question for g++-multilib gcc-multilib.